packages/web/src/components/search/addons/Mic.js

```diff
@@ -129,7 +129,9 @@ class Mic extends React.Component {
 		const { className } = this.props;
 		switch (status) {
 			case STATUS.active:
-				return <ListenSvg className={className} onClick={this.handleClick} />;
+				return (
+					<ListenSvg className={`${className} --listen-svg`} onClick={this.handleClick} />
```

Comment:
Why is space b/w `className` and `--listen-svg`?

Reply:
because they are different classes, one is an internal class(--internal-svg) and the other one is class passed by the suer

Reply:
Use `getClassName` function